generators/chipyard/src/main/scala/config/NoCConfigs.scala

```diff
@@ -62,19 +63,19 @@ import scala.collection.immutable.ListMap
  */
 // DOC include start: MultiNoCConfig
 class MultiNoCConfig extends Config(
-  new constellation.soc.WithCbusNoC(constellation.protocol.TLNoCParams(
+  new constellation.soc.WithCbusNoC(constellation.protocol.SimpleTLNoCParams(
```

Comment:
where does SimpleTLNoC reside? hopefully, there are comments there

Reply:
The `constellation.protocol.SimpleTLNoCParams` should indicate it lives in the Constellation repo.

Reply:
Constellation repo has adequate comments which can infer use case. Approved